src/main/groovy/io/gatling/gradle/GatlingPlugin.groovy

```diff
@@ -86,7 +87,18 @@ class GatlingPlugin implements Plugin<Project> {
                     gatlingImplementation project.sourceSets.test.output
                 }
             }
+
+            if (project.repositories.isEmpty() || isGatlingUnresolved(project)) {
```

Comment:
When Gatling is unresolved, shouldn't you be adding mavenCentral to the existing repositories instead (might not be empty)?

Reply:
I didn't understand this comment :(

Reply:
I assume that `project.repositories {  mavenCentral() }` ditches all user-defined repositories, right?

What if the user has configured an internal repository that doesn't contain the Gatling libs? Shouldn't you append mavenCentral to the existing repositories list instead of dropping the existing one(s)?

Reply:
Well, your assumption is wrong. Take a look at provided test `should add mavenCentral if repo with gatling not declared` to see the proof :)

Reply:
I stand corrected. Thanks.

Groovy syntax never ends to puzzle me...

Reply:
It's not Groovy in this particular case, but Gradle DSL.

This method is called behind the scene in class `org.gradle.api.internal.project.DefaultProject`

```
    public void repositories(Closure configureClosure) {
        ConfigureUtil.configure(configureClosure, this.getRepositories());
    }
```